prometheus-ksonnet/lib/prometheus-config.libsonnet

```diff
@@ -171,6 +171,17 @@
             replacement: '__param_$1',
           },
 
+          // Generic mapping of k8s label/annotation to prometheus labels.
+          {
+            regex: '__meta_kubernetes_pod_label_prometheus_io_label_(.+)',
```

Comment:
I found a request on the mailing list and It didn't include the`replacement` field.
https://github.com/prometheus/prometheus/blob/86426c05669417cf1012fbc3a40fb7e0f5a8a101/documentation/examples/prometheus-kubernetes.yml#L113-L114

Reply:
Indeed. That's because `$1` is the default value for `replacement`. \o/